apps/src/templates/rubrics/ProgressRing.jsx

```diff
@@ -4,15 +4,17 @@ import i18n from '@cdo/locale';
 import style from './rubrics.module.scss';
 import {learningGoalShape} from './rubricShapes';
 import color from '@cdo/apps/util/color';
+import classnames from 'classnames';
 
 export default function ProgressRing({
+  className,
   learningGoals,
   currentLearningGoal,
   understandingLevels,
   radius,
   stroke,
 }) {
-  const normalizedRadius = radius - stroke * 2;
+  const normalizedRadius = radius - stroke;
```

Comment:
You only need to account for the width of the stroke once since this is the radius and not the diameter and the stroke radiates from the outline of the circle. So there's only one half of it extended beyond the radius.